src/core/server/saved_objects/import/validate_data_sources.ts

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import { fetchNestedDependencies } from '../export/inject_nested_depdendencies';
+import { SavedObject, SavedObjectsBaseOptions, SavedObjectsClientContract } from '../types';
+import { SavedObjectsImportError } from './types';
+import { findReferenceDataSourceForObject } from './utils';
+
+// Check whether the target workspace includes the data source from the references of coping assets.
```

Comment:
This function doesn't mention anything related to `coping assets`, I suggest to tweaks a bit the comment:
```suggestion
// Check whether the target workspace includes the data source referenced by the savedObjects
```

Reply:
updated